hack/run-user-cluster-controller-manager.sh

```diff
@@ -20,13 +20,14 @@ cd $(dirname $0)/..
 source hack/lib.sh
 
 KUBERMATIC_DEBUG=${KUBERMATIC_DEBUG:-true}
+PPROF_PORT=${PPROF_PORT:-6600}
```

Comment:
port 6600 is already occupied :D

```shell
$ grep PPROF_PORT hack/*.sh
hack/run-api.sh:PPROF_PORT=${PPROF_PORT:-6600}
hack/run-api.sh:  -pprof-listen-address=":$PPROF_PORT" \
hack/run-master-controller-manager.sh:PPROF_PORT=${PPROF_PORT:-6600}
hack/run-master-controller-manager.sh:  -pprof-listen-address=":${PPROF_PORT}" \
hack/run-seed-controller-manager.sh:PPROF_PORT=${PPROF_PORT:-6600}
hack/run-seed-controller-manager.sh:  -pprof-listen-address=":${PPROF_PORT}" \
hack/run-user-cluster-controller-manager.sh:PPROF_PORT=${PPROF_PORT:-6600}
hack/run-user-cluster-controller-manager.sh:  -pprof-listen-address=":${PPROF_PORT}" \
```

Reply:
And that is exactly why this line is there. So you can override it.